src/App.vue

```diff
@@ -57,4 +57,10 @@ export default {
 #nav a.router-link-exact-active {
   color: #42b983;
 }
+
+@font-face {
+  font-family: "Kruti Dev";
+  src: local("Kruti Dev"),
+    url('./fonts/Kruti_Dev_10.TTF') format("truetype");
```

Comment:
Installing the Kruti font. Used by CMS for typing in hindi.

Reply:
Does this work after deploying? Maybe we should put the font on s3, make it public and include the URL here. Or use a CDN or something

Reply:
Does this work after deploying also, since it's a local file? Might be better to store it on S3 or somewhere

Reply:
It works after deploying as the font is packaged along with the frontend files and hosted on S3. The only benefit of keeping the font on S3 at some location would be that the size of the static web files will be much smaller

Reply:
Much? Font is that big kya? 

Might make sense to keep it in s3 and load it only when needed, then?

Reply:
Today is Thursday lol. Deploy it off if the size of the font isn't crazy